Chimera doesn't have `shred`, nor does BSD, and it's usefulness is questionable, `rm -P` should give the same
 functionality but as the man page describes `This flag has no effect.`. So just delete the whole shredding part.

password-store also assumes a util-linux getopt, so force it to use ugetopt

diff --git a/src/password-store.sh b/src/password-store.sh.new
index 7a26d7fa..c46ef246 100755
--- a/src/password-store.sh
+++ b/src/password-store.sh.new
@@ -234,15 +234,13 @@ tmpdir() {
 		)"
 		SECURE_TMPDIR="$(mktemp -d "${TMPDIR:-/tmp}/$template")"
 		shred_tmpfile() {
-			find "$SECURE_TMPDIR" -type f -exec $SHRED {} +
 			rm -rf "$SECURE_TMPDIR"
 		}
 		trap shred_tmpfile EXIT
 	fi
 
 }
-GETOPT="getopt"
-SHRED="shred -f -z"
+GETOPT="ugetopt"
 BASE64="base64"
 
 source "$(dirname "$0")/platform/$(uname | cut -d _ -f 1 | tr '[:upper:]' '[:lower:]').sh" 2>/dev/null # PLATFORM_FUNCTION_FILE
